src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/TypeImportCompletionCacheEntry.cs

```diff
@@ -159,11 +159,21 @@ public TypeImportCompletionCacheEntry ToReferenceCacheEntry()
 
             public void AddItem(INamedTypeSymbol symbol, string containingNamespace, bool isPublic)
             {
-                // We want to cache items with EditoBrowsableState == Advanced regardless of current "hide adv members" option value
-                var (isBrowsable, isEditorBrowsableStateAdvanced) = symbol.IsEditorBrowsableWithState(
-                    hideAdvancedMembers: false,
-                    _editorBrowsableInfo.Compilation,
-                    _editorBrowsableInfo);
+                bool isBrowsable, isEditorBrowsableStateAdvanced;
+                try
+                {
+                    // We want to cache items with EditorBrowsableState == Advanced regardless of current "hide adv members" option value
+                    (isBrowsable, isEditorBrowsableStateAdvanced) = symbol.IsEditorBrowsableWithState(
+                        hideAdvancedMembers: false,
+                        _editorBrowsableInfo.Compilation,
+                        _editorBrowsableInfo);
+                }
+                catch (Exception)
```

Comment:
Should we restrict to `Microsoft.CodeAnalysis.UnsupportedSignatureContent` here? Just to make sure we report other problems? 

I could see a catch all potentially hiding NFW

Reply:
Yea that'd be better but[ `UnsupportedSignatureContent` ](https://github.com/dotnet/roslyn/blob/main/src/Compilers/Core/Portable/MetadataReader/UnsupportedSignatureContent.cs)is internal to compiler.

Reply:
Actually, the exception is already handled by metadata reader and never leaked out. Seems like this is just a case of first chance expectation being flagged by RPS. I will close this and consider the bug as by-design.

https://github.com/dotnet/roslyn/blob/main/src/Compilers/Core/Portable/MetadataReader/MetadataDecoder.cs#L1802